packages/headless/src/controllers/results-per-page/headless-results-per-page.ts

```diff
@@ -56,24 +62,26 @@ export function buildResultsPerPage(
     ...controller,
 
     /**
-     * @returns The state of the `ResultsPerPage` controller.
+     * The state of the `ResultsPerPage` controller.
      */
     get state() {
       return {
+        /** The number of results per page. */
         numberOfResults: engine.state.pagination.numberOfResults,
       };
     },
 
-    /** Updates the number of results to request
-     * @param number The number of results.
+    /** Updates the number of results to request per page.
+     * @param num The number of results.
      */
     set(num: number) {
       dispatch(updateNumberOfResults(num));
       dispatch(executeSearch(logPagerResize()));
     },
 
-    /** Returns `true` if the number of results is equal to the passed value, and `false` otherwise.
-     * @returns boolean
+    /** Checks whether the number of results per page is equal to the specified number.
+     * @param num The number of results.
+     * @returns Is `true` if the number of results is equal to the passed value, and `false` otherwise.
```

Comment:
I usually consider "returns" to be the first word of this sentence, so:
```suggestion
     * @returns `true` if the number of results is equal to the passed value, and `false` otherwise.
```